README.md

```diff
@@ -0,0 +1,18 @@
+# Lazy
+
+A library for lazy objects, started life as a coredata/contrib package.
```

Comment:
Not a blocker for this PR, but an example of what these lazy objects are might help down the line. Since this line describes where it came from, but it adds little description of what it is.